### DISCUSSION TITLE
Join .NET Foundation

### DISCUSSION THREAD

**Discussion by xivk** [NetTopologySuite/NetTopologySuite#547] [Ideas]
Not sure what the other maintainers here think but would it be a good idea to submit NTS as a project to the dotnet foundation?

https://dotnetfoundation.org/projects/submit

**Comment by airbreather** [comment]
We had a bit of a short-lived discussion about this a few years back (I can't find it now -- I thought it was in one of those private groups, but I don't see it there now).  All I can find is [my reply](https://github.com/NetTopologySuite/NetTopologySuite/issues/237#issuecomment-389162604) that has some initial thoughts:

> > I suggested to join the .NET foundation
> 
> One reservation I have about this move is that joining the .NET Foundation requires adopting [their Code of Conduct](https://dotnetfoundation.org/code-of-conduct), which contains the following lines:
> 
> > Project maintainers who do not follow or enforce the Code of Conduct may be permanently removed from the project team.
> > This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the project or its community.
> 
> As far as I can tell, that means that the .NET Foundation could, at their sole judgement, determine that a project maintainer is acting "unethically" or "unprofessionally" (their words, and their judgement) and remove that maintainer's privileges, even if that person was out and about, just because the .NET Foundation arbitrarily deemed that the person was "representing the project or its community" at the time.
> 
> Not that I'm completely against it or anything, I just don't see enough benefit at this point in time to recommend that course of action.

The current version of the Code of Conduct does still seem to imply the same basic effect that I was concerned about before, but it isn't positioned prominently within the document, which is a welcome improvement IMO.

Plus, the [benefits](https://dotnetfoundation.org/projects/why-join) do seem to include more things than before, and so now I'm genuinely on the fence about it.  I've also been spending **much** less of my time on NTS recently, so I figure that my opinions matter a lot less these days :grinning:.
